amr-wind/incflo.cpp

```diff
@@ -372,6 +378,10 @@ void incflo::init_physics_and_pde()
         if (activate_overset) {
             m_sim.activate_overset();
         }
+
+        // Get number of advection iterations
+        pp.query("advection_iterations", m_adv_iters);
```

Comment:
and when we update the naming, make sure the user facing input makes sense within that context as well.